### ISSUE TITLE
README does not state how to retrieve most recent version of go-ipfs through IPFS

### ISSUE THREAD

**Issue by yarwelp** [ipfs/go-ipfs#1952]

**Comment by whyrusleeping** [comment]
This would definitely be good docs to have. I'll add a todo for this on our documentation sprint repo

**Comment by whyrusleeping** [comment]
For anyone who wants to write the docs for this, The way to check the latest version of ipfs  *with* ipfs is:
```
ipfs cat /ipns/dist.ipfs.io/go-ipfs/versions
```
Then, to view available builds for a version:
```
ipfs ls /ipns/dist.ipfs.io/go-ipfs/$VERSION
```
Then, to download a given build of a version:
```
ipfs get /ipns/dist.ipfs.io/go-ipfs/$VERSION/go-ipfs_$VERSION_linux-arm.tar.gz #for the arm linux bin
```

**Comment by leerspace** [comment]
I think this one was fixed with #4390.

**Comment by whyrusleeping** [comment]
Yep, Thanks @leerspace!
